gateway/ln-gateway/src/events.rs

```diff
@@ -0,0 +1,51 @@
+use fedimint_client::db::event_log::{Event, EventKind};
+use fedimint_core::core::ModuleKind;
+use fedimint_core::Amount;
+use fedimint_lnv2_common::contracts::{Commitment, OutgoingContract};
+use serde::{Deserialize, Serialize};
+
+use crate::sha256::Hash;
+
+#[derive(Serialize, Deserialize)]
+pub struct OutgoingPaymentSuccess {
+    pub outgoing_contract: OutgoingContract,
+    pub min_contract_amount: Amount,
+    pub invoice_amount: Amount,
+    pub start_secs: u64,
+    pub end_secs: u64,
+    pub max_delay: u64,
```

Comment:
Seconds do not seem finegrained enough for latency tracking. How about Instant?

Reply:
`Instant` also does not preserve the absolute time that the transaction happened, which I think is valuable